maneuvering_task_node:
  ros__parameters:
    map_origin_lat: -33.72213988382845
    map_origin_lon: 150.67413500672993
    map_origin_set: true
    gps_start_lat: 63.44097054434422
    gps_start_lon: 10.419997767413607
    gps_end_lat: 63.44125901804796
    gps_end_lon: 10.41857835889424
    gps_start_x: 0.0
    gps_start_y: 0.0
    gps_end_x: 141.45540473589617
    gps_end_y: -0.22386536008730218
    gps_frame_coords_set: true
    map_origin_topic: "/map/origin"
    odom_topic: "/seapath/odom/ned"
    landmark_topic: "landmarks_out"
    assignment_confidence: 10  # Number of consequtive identical assignments from auction algorithm before we consider the assignment as correct
    
    # Task specific parameters
    distance_to_first_buoy_pair: 2.0 # Distance in x-direction to first buoy pair in meters from current postition (not gps) position in base_link frame.
    # If the distance is greater than five we drive distance-3m towards end before trying to find the first buoy pair
    initial_offset: 1.0 # The vertical offset from the second and third buoy pair in meters
    nr_of_pair_in_formation: 20 # Number of buoy pairs in formation, excluding the start and end gate at gps points (NB! pdf is not accurate)
    